### ISSUE TITLE
How to get the reported result?

### ISSUE THREAD

**Issue by lucyboll** [namisan/mt-dnn#18]
Thanks for your work and we have a problem with the reproduce. We use the released pretrained model "mt_dnn_large.pt" and directly run "run_stsb.sh", "run_rte" and so on, however, we get low accuracy. I want to know is there any problem with the running procedure or the released model? Or something else such as the number of GPU and bach size?

**Comment by namisan** [comment]
We use 8 V100 with a batch size of 32.


Attached are log.
_03/10/2019 07:00:14 Total number of params: 335143938
03/10/2019 07:00:18 At epoch 0
03/10/2019 07:00:31 Task [ 0] updates[     1] train loss[0.79459] remaining[0:34:31]
03/10/2019 07:03:26 Task rte -- epoch 0 -- Dev ACC: 81.588
03/10/2019 07:03:50 [new test scores saved.]
03/10/2019 07:03:54 At epoch 1
03/10/2019 07:06:48 Task rte -- epoch 1 -- Dev ACC: 81.949
03/10/2019 07:07:12 [new test scores saved.]
03/10/2019 07:07:15 At epoch 2
03/10/2019 07:10:10 Task rte -- epoch 2 -- Dev ACC: 83.394
03/10/2019 07:10:34 [new test scores saved.]
03/10/2019 07:10:37 At epoch 3
03/10/2019 07:11:12 Task [ 0] updates[   500] train loss[0.27226] remaining[0:02:16]
03/10/2019 07:13:31 Task rte -- epoch 3 -- Dev ACC: 83.394
03/10/2019 07:13:56 [new test scores saved.]
03/10/2019 07:13:59 At epoch 4
03/10/2019 07:16:54 Task rte -- epoch 4 -- Dev ACC: 83.755
03/10/2019 07:17:18 [new test scores saved.]_

**Comment by KushajveerSingh** [comment]
In case we do not have that much GPU RAM, can we use gradient accumulation to increase our batch size in pytorch.

**Comment by Rajratnpranesh** [comment]
@KushajveerSingh Are you using CUDA with Pytorch? If yes, please let me know the versions of both. Thank you so much.

**Comment by namisan** [comment]
We support gradient accumulation now.
